app/routes/questionnaire.py

```diff
@@ -406,17 +406,14 @@ def get_view_submission(schema):
                 metadata,
             )
 
-            summary_rendered_context = summary_context.build_all_groups()
-
-            context = {
-                "summary": {
-                    "groups": summary_rendered_context,
-                    "answers_are_editable": False,
-                    "is_view_submission_response_enabled": is_view_submitted_response_enabled(
-                        schema.json
-                    ),
-                }
-            }
+            is_view_submission_response_enabled = is_view_submitted_response_enabled(
+                schema.json
+            )
+            context = final_summary_context(
+                answers_are_editable=False,
+                is_view_submission_response_enabled=is_view_submission_response_enabled,
+            )
+            del context["summary"]["summary_type"]
```

Comment:
It's was needed for the summary context template, but broke the view submission (which uses the summary partial template). Now that I have changed the templates, this may not be required. I'll retest without the delete :+1:

Reply:
Tested without the delete... It's no longer needed